spec/classes/repo_spec.rb

```diff
@@ -18,6 +18,21 @@
           })
         }
       end
+      context 'sensu::repo apt', :if => facts['osfamily'] == 'Debian' do
+        it {
+          should contain_apt__source('sensu_beta').with({
```

Comment:
don't we need apt in the .fixtures.yml then

Reply:
Was already in .fixutres.yml.  The repo specs are not actually running, I need to fix them tonight.  I think the fact keys end up being symbols in unit tests with rspec-puppet-facts, at least that's my theory.  Doesn't affect manifest code, just effects how facts are referenced in tests.